packages/connections/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@dcos/connections",
+  "main": "index.js",
+  "version": "0.1.0",
+  "license": "Apache-2.0",
+  "scripts": {
+    "test": "jest"
+  },
+  "devDependencies": {
+    "jest": "19.0.2",
+    "jest-cli": "16.0.2"
+  }
+}
```

Comment:
Those `lerna.js` folks specify a main repo as a package repo in `package.json` we might want to follow that as well.

Reply:
<img src="https://cloud.githubusercontent.com/assets/647035/21101001/3a24fe0a-c077-11e6-810d-95006b39fa85.gif" width="100%" />